pkg/reconciler/revision/resources/deploy.go

```diff
@@ -227,6 +291,9 @@ func MakeDeployment(rev *v1alpha1.Revision,
 			podTemplateAnnotations[IstioOutboundIPRangeAnnotation] = networkConfig.IstioOutboundIPRanges
 		}
 	}
+	if err := verifySecrets(rev, secretLister); err != nil {
```

Comment:
Sorry to be clear I was thinking _after_ MakeDeployment and _in_ CreateDeployment before the actual create `	return c.KubeClientSet.AppsV1().Deployments(deployment.Namespace).Create(deployment)`

Reply:
Ah I see. Thank you for the clarification!